appinventor/components/src/com/google/appinventor/components/scripts/ComponentProcessor.java

```diff
@@ -1386,8 +1392,8 @@ private void processProperties(ComponentInfo componentInfo,
             } else if (priorProperty.writable) {
               // TODO(user): handle lang_def and document generation for multiple setters.
               throw new RuntimeException("Inconsistent types " + priorProperty.type +
-                                         " and " + newProperty.type + " for property " +
-                                         propertyName + " in component " + componentInfo.name);
+                                        " and " + newProperty.type + " for property " +
```

Comment:
Looks like your editor played games with the indentation in this file. Correct the lines with whitespace only changes to reflect what's in the base branch.